src/Microsoft.AspNetCore.Mvc.Core/project.json

```diff
@@ -20,22 +20,22 @@
     "xmlDoc": true
   },
   "dependencies": {
-    "Microsoft.AspNetCore.Authorization": "1.0.1",
-    "Microsoft.AspNetCore.Hosting.Abstractions": "1.0.1",
-    "Microsoft.AspNetCore.Mvc.Abstractions": "1.0.2",
-    "Microsoft.AspNetCore.Http": "1.0.1",
-    "Microsoft.AspNetCore.Routing": "1.0.2",
+    "Microsoft.AspNetCore.Authorization": "1.0.2",
+    "Microsoft.AspNetCore.Hosting.Abstractions": "1.0.2",
+    "Microsoft.AspNetCore.Mvc.Abstractions": "1.0.3",
+    "Microsoft.AspNetCore.Http": "1.0.2",
+    "Microsoft.AspNetCore.Routing": "1.0.3",
     "Microsoft.AspNetCore.Routing.DecisionTree.Sources": {
       "type": "build",
-      "version": "1.0.2"
+      "version": "1.0.3"
```

Comment:
> Any particular concerns regarding sources packages?

No. I just noticed this was the only .Sources dependency you updated. Your answer makes sense.